common/analysis/lint_waiver.cc

```diff
@@ -170,6 +197,9 @@ void LintWaiverBuilder::ProcessTokenRangesByLine(
     ProcessLine(token_range, i);
   }
 
+  // Apply regex waivers
+  lint_waiver_.RegexToLines(text_structure.Contents());
```

Comment:
Should I compute it all the way back in `VerilogLintTextStructure` and change the interface of all the relevant functions in between? Adding it to the class does not seem like a good option because it doesn't have the content of the file either.

Reply:
It is already available by the time you enter `VerilogLintTextStructure()`, so yes, you could pass it along through functions, as done here:

https://github.com/google/verible/blob/8177dd80cff4ba2ae884451e6c5316edb2b4dbe7/verilog/analysis/verilog_linter.cc#L253

And you're right that it should always be passed along with the content of the file or something that carries the content, like `TextStructureView`.

Reply:
Absolutely right, I misunderstood what you meant the first time, thanks, fixed